deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -2872,7 +2819,7 @@ spec:
               recurringJobs:
                 description: Deprecated. Replaced by a separate resource named "RecurringJob"
                 items:
-                  description: 'VolumeRecurringJobSpec is a deprecated struct. TODO: Should be removed when recurringJobs gets removed from the volume       spec.'
+                  description: 'VolumeRecurringJobSpec is a deprecated struct. TODO: Should be removed when recurringJobs gets removed from the volume spec.'
```

Comment:
Thanks!

Reply:
Looks like this is fixed in controller-gen v0.8.0. Do we want to fix this in https://github.com/longhorn/longhorn-manager/blob/97460182360e925fddfc9e586814f63b2b6a1344/k8s/pkg/apis/longhorn/v1beta2/volume.go#L104